cli/azd/pkg/input/console.go

```diff
@@ -149,24 +151,43 @@ func (c *AskerConsole) MessageUxItem(ctx context.Context, item ux.UxItem) {
 	}
 }
 
+func (c *AskerConsole) spinnerText(title, charset string) string {
+
+	spinnerLen := len(charset) + 1 // adding one for the empty space before the message
+
+	if len(title)+spinnerLen >= c.consoleWidth {
```

Comment:
Don't we need to account for `len(postfix)` here?

Reply:
no. postfix replaces the last 3 characters from `spinner + message` when and only when there is overflow.

We don't append the postfix to the message, we replace the ending of the message after removing the overflow from it